atef/widgets/ophyd.py

```diff
@@ -611,17 +615,24 @@ def copy(*_):
             def select_attr(*_):
                 self.attributes_selected.emit([row_data])
 
+            standard_actions = []
+
             if index.data() is not None:
-                copy_action = self.menu.addAction(f"&Copy: {index.data()}")
+                copy_action = QtWidgets.QAction(f'&Copy: {index.data()}')
+                standard_actions.append(copy_action)
                 copy_action.triggered.connect(copy)
 
             row_data = self.get_data_from_proxy_index(index)
             if row_data is not None:
-                select_action = self.menu.addAction(
+                select_action = QtWidgets.QAction(
                     f"&Select attribute: {row_data.attr}"
```

Comment:
Can somebody remind me what the function of this "Select attribiute" action was supposed to do? It doesn't seem to do anything on master or here on this branch.

Reply:
It should select the attribute and add it to the atef config attribute list. It's not so obvious because it happens in the background - assuming it is still working...

Reply:
It doesn't seem to do this any more. I need to click the "select" button at the bottom or one of the new actions Jeremy added here to add to the config.

Reply:
Confirmed that it doesn't actually work. Looks like the slot wasn't connected somewhere along the way.